src/models/subgraph-entities/project.ts

```diff
@@ -145,3 +145,102 @@ export const parseTrendingProjectJson = (
   trendingScore: BigNumber.from(project.trendingScore),
   trendingVolume: BigNumber.from(project.trendingVolume),
 })
+
+type BaseProjectVx = {
+  id: string
```

Comment:
Project `id` is no longer just a number. We likely won't use this for anything in the UI https://github.com/jbx-protocol/juice-subgraph/blob/3031de344c1125e222a31eff0503648e508a891e/src/utils.ts#L18